src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
```

Comment:
4. Error 4: The `subtract` method has an extra parameter `c` that is not used. This extra parameter should be removed to avoid confusion.